### PR TITLE
Add support for WiX installer

### PR DESCRIPTION
**Pull request by cjh1**
Here is the initial pass at adding WiX support. 

One improvement we can make in the future would be to override the default dialog graphic with something more tomviz specific.

### CODE REVIEW

**Comment by cjh1** [comment]
@cryos @mathturtle PTAL

File: projects/win32/tomviz.bundle.cmake [hunk 1]
```diff
@@ -54,15 +58,8 @@ install(DIRECTORY "${install_location}/lib/tomviz"
 if(itk_ENABLED)
 install(DIRECTORY "${install_location}/lib/itk"
         DESTINATION "lib"
-	USE_SOURCE_PERMISSIONS
-	COMPONENT ${AppName})
-endif()
-
-#------------------------------------------------------------------------------
-set (CPACK_NSIS_MUI_ICON "${CMAKE_CURRENT_LIST_DIR}/InstallerIcon.ico")
```
Comment (cryos) [inline projects/win32/tomviz.bundle.cmake]: Are these no longer needed for NSIS, or were they removed in error?

Comment (cjh1) [inline projects/win32/tomviz.bundle.cmake]: They have been moved to the NSIS specific part of [cpack_config_file.cmake.in](https://github.com/cjh1/tomviz-superbuild/blob/wix-support/projects/win32/cpack_config_file.cmake.in#L7-L11)

**Comment by cryos** [comment]
Sounds good, I am assuming you tested this, nothing jumps out to me as an issue. Let's merge it and see how the builders do.

**Pull request merged**